packages/web-components/src/components/va-file-input/va-file-input.scss

```diff
@@ -210,3 +210,11 @@
   color: var(--vads-color-secondary-dark);
   margin-left: 0.25rem;
 }
+
+h1, h2, h3, h4, h5, h6 {
+  .required {
+    font-family: "Source Sans Pro Web", "Source Sans Pro", "Helvetica Neue", Helvetica, Arial, sans;
```

Comment:
Similar thing here, could we use our tokens? https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/css-library/dist/tokens/css/variables.css#L127